R/g.part6.R

```diff
@@ -29,6 +29,7 @@ g.part6 = function(datadir = c(), metadatadir = c(), f0 = c(), f1 = c(),
   # Identify correct subfolder
   expected_ts_path = paste0(metadatadir, "/meta/ms5.outraw/", params_phyact[["part6_threshold_combi"]])
   expected_ms5raw_path = paste0(metadatadir, "/meta/ms5.outraw")
+  expected_ms5_path = paste0(metadatadir, "/meta/ms5.out")
```

Comment:
Please undo all changes to part 6. The same result can also be achieved by adding the filename as object to the part 5 time series files (RData) and using that in part 6.

For those who use csv exports in part 5 this will not work and they have to find their own solution by matching filenames between the csv reports. The added code complexity and extra file loading effort in your proposed solution are I think not worth it given that this concerns only such a niche use-case.

Reply:
Ok, so the `RData` time series in part 5 would be a list consisting of 2 objects: the `filename` and the `mdat` data frame, is that what you mean? We just need to unlist the mdat in part 6 in order to use it.

Reply:
It does not have to be a list, `save` can carry multiple objects. For example save(A, B, C, file = "C:/myfile.RData")
will save both object A and B without the need to list them. We do that already in lots of places in GGIR.